full-stack-tests/rpc-interface/src/frontend/setup/TestContext.ts

```diff
@@ -68,10 +67,22 @@ export class TestContext {
         clientId: this.settings.oidcClientId,
```

Comment:
I readded the adminUserAccessToken - start by replacing the usage of this browser adminUserAccessToken with your serviceAuthToken, one test file at a time. Once you've removed all usage of the admin user token, you can then remove it entirely from the rpc interface's full stack test

Reply:
> is the adminUserToken used anywhere in anymore in rpc-interface's tests? If not, why not remove the variable and initialization of it entirely from TestContext?

sounds good, just did